### ISSUE TITLE
Keys remain pressed when player completes level

### ISSUE THREAD

**Issue by evndean** [evndean/link-runner#46]
When player input gets removed at level completion (see #31), if a player was holding down a key when they finished the level, the key remains pressed. For example, if the player was holding down the space bar to shoot when they finished the level, the drone continues to shoot until the player moves to the next level; if the player was holding the right arrow key when they finished the level, the drone continues to move right until the player moves to the next level.

**Comment by evndean** [comment]
This was resolved in #53. Will close this once the current develop branch is merged into master.
